Mathlib/Probability/Density.lean

```diff
@@ -411,6 +424,35 @@ end IsUniform
 
 end
 
+section TwoVariables
+
+open ProbabilityTheory
+
+variable {Ω E F : Type*} [MeasurableSpace Ω] [MeasurableSpace E] [MeasurableSpace F]
+variable {X : Ω → E} {Y : Ω → F} {ℙ : Measure Ω} {μ : Measure E} {ν : Measure F}
+variable [HasPDF (fun ω ↦ (X ω, Y ω)) ℙ (μ.prod ν)]
```

Comment:
Most of these are already declared (you can use `#where` to see the variables that exist at any point).
```suggestion
variable {F : Type*} [MeasurableSpace F] {ν : Measure F} {X : Ω → E} {Y : Ω → F}
```